.env.example

```
@@ -0,0 +1,42 @@
+NEXT_PUBLIC_SITE_URL=https://brandonperfetti.com
+
+# --- Notion CMS (required for CMS_PROVIDER=notion) ---
+NOTION_API_TOKEN=your_notion_integration_token
+NOTION_API_VERSION=2025-09-03
+NOTION_CMS_ARTICLES_DATA_SOURCE=collection://replace-with-id
+NOTION_CMS_PAGES_DATA_SOURCE=collection://replace-with-id
+NOTION_CMS_PROJECTS_DATA_SOURCE=collection://replace-with-id
+NOTION_CMS_TECH_DATA_SOURCE=collection://replace-with-id
+NOTION_CMS_USES_DATA_SOURCE=collection://replace-with-id
+NOTION_CMS_WORK_HISTORY_DATA_SOURCE=collection://replace-with-id
+NOTION_CMS_AUTHORS_DATA_SOURCE=collection://replace-with-id
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NOTION_CMS_AUTHORS_DATA_SOURCE key should go before the NOTION_CMS_PAGES_DATA_SOURCE key

(UnorderedKey)